test/test_cpp_extensions_jit.py

```diff
@@ -885,29 +903,31 @@ def run_test(stderr_file, destination):
             }}
             """)
 
-            # Some special stuff to overwrite stderr for a C++ extension
-            # Copied from: https://stackoverflow.com/questions/8804893/redirect-stdout-from-python-for-c-calls
-            sys.stdout.flush()
-            newstdout = os.dup(2)
-            devnull = os.open(stderr_file, os.O_WRONLY)
-            os.dup2(devnull, 2)
-            os.close(devnull)
-            sys.stdout = os.fdopen(newstdout, 'w')
-
-            module = torch.utils.cpp_extension.load_inline(
-                name="segfault",
-                cpp_sources=csrc,
-                functions=["fail"],
-            )
-            module.fail()
+        # Some special stuff to overwrite stderr for a C++ extension
+        # Copied from: https://stackoverflow.com/questions/8804893/redirect-stdout-from-python-for-c-calls
+        sys.stdout.flush()
+        newstdout = os.dup(2)
+        devnull = os.open(stderr_file, os.O_WRONLY)
+        os.dup2(devnull, 2)
+        os.close(devnull)
+        sys.stdout = os.fdopen(newstdout, 'w')
 
+        module = torch.utils.cpp_extension.load_inline(
+            name="segfault",
+            cpp_sources=csrc,
+            functions=["fail"],
+        )
+        module.fail()
 
-        with tempfile.TemporaryDirectory() as temp_dir, tempfile.NamedTemporaryFile() as stderr:
+    @unittest.skipIf(TEST_WITH_ASAN, "ASAN disables the crash handler's signal handler")
```

Comment:
Why are you removing `has_breakpad` option, as devs can still choose to compile pytorch without it.